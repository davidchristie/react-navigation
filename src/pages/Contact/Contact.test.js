import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import React from 'react'

import Contact from './Contact'

it('renders correctly', () => {
  expect(toJson(shallow(
    <Contact />
  ))).toMatchSnapshot()
})
