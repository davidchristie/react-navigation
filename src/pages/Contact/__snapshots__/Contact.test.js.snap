// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Contact
  </h1>
</div>
`;
